lib/tasks/va_facilities.rake

```diff
@@ -68,7 +61,228 @@ namespace :va_facilities do
         end
       end
     end
+    puts "All VA facilities have been created or updated in the DB!"
```

Comment:
this should just say "All VA facilities have been created in the DB!" since we technically aren't updating anything

Reply:
No we are... if the facility exists we update.

Reply:
in this particular task we are only creating... I am wondering if we need this task `:create_va_facilities_and_transfer_data` and we can just rename `:update_va_facilities` to be both `:create_or_update_va_facilities` since that's what it is doing

Reply:
Yeah - the first task 'create_va_facilities_and_transfer_data" is obsolete...  theis is the original task Josh wrote for the first import.  I'll remove that to avoid confusion.  I renamed the task we need to run for this PR to create_or_update_va_facilities.  We should be able to re-use this code for future imports (unless they make changes to the schema of course).